zellij-server/src/screen.rs

```diff
@@ -863,394 +896,186 @@ pub(crate) fn screen_thread_main(
                         }
                     }
                 };
-                screen
-                    .bus
-                    .senders
-                    .send_to_server(ServerInstruction::UnblockInputThread)
-                    .unwrap();
+                screen.unblock_input();
                 screen.update_tabs();
-
-                screen.render();
+                continue;
```

Comment:
> Also I read in one of the previously existent comments that screen.update_tabs() calls screen.render() eventually, which is why I added a continue after every update_tabs(). Is this really the case or has this changed in the meantime?

As a result of simply checking the internal logic of `update_tabs` and `render`, I think the two purposes are different. But I'm not sure. @imsnif Could you give me your opinion on this part?